src/components/PostList.server.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import getPosts from "../server/posts";
+import Link from "../framework/Link";
+
+export default async function PostList() {
+  const list = await getPosts();
```

Comment:
the Server part of the original component, wee we can use async await here without API endpoints